src/js/components/reviewData/CertificationModal/RevalidateDataModal.jsx

```diff
@@ -31,15 +31,14 @@ export default class ReviewDataModal extends React.Component {
                 hashHistory.push('/validateData/' + this.props.submissionID);
             })
             .catch((error) => {
+                let errMsg = '';
```

Comment:
Since this only gets hit when there definitely is an error, you could set it to the message on line 38 as default. that would remove the else statement but have the same behavior. This is just preference though